build.sbt

```diff
@@ -13,6 +13,7 @@ ThisBuild / apacheSonatypeProjectProfile := "pekko"
 sourceDistName := "apache-pekko-connectors"
 sourceDistIncubating := true
 ThisBuild / resolvers += Resolver.ApacheMavenSnapshotsRepo
+ThisBuild / resolvers += Resolver.ApacheMavenStagingRepo
```

Comment:
I think this is left over from an earlier point in time, we rebase against`main`?

Reply:
removed and rebased